### PR TITLE
Bryce/localstack snowflake ci poc

### PR DESCRIPTION
**Pull request by bcodell**
This PR:
* adds CI for Snowflake using Localstack's Snowflake extension
* adds a sponsor section to the readme

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -35,6 +38,18 @@ jobs:
         env:
           GCP_KEYFILE_PATH: ./gcp_keyfile.json
           DBT_PROFILES_DIR: . # Use integration_tests/profiles.yml
+      - name: dbt CI - snowflake
+        run: |
+          localstack start -d
+          localstack extensions install localstack-extension-snowflake
```
Comment (whummer) [inline .github/workflows/ci.yml:44]: @bcodell can we reverse the order of commands here - running the `extensions install` command first, then starting up the container? 

I know it's a bit counter-intuitive, but the LocalStack container is actually not required for the extension installation (this is managed by the CLI command automatically), and the container actually needs to be started after the extension has been installed (to initialize the extension on startup).

Alternatively, we could set the `EXTENSION_AUTO_INSTALL=localstack-extension-snowflake` environment variable, then the extension will be automatically installed on container startup (i.e., no need to install it via the CLI).

Comment (bcodell) [reply .github/workflows/ci.yml:44]: Works like a charm now - thanks for the assist! 🙏

**Comment by bcodell** [comment]
@whummer this is just about ready for deployment! I added a [sponsor section](https://github.com/bcodell/dbt-aql/blob/5fb2a3cbf489e95a5e057e3904807afa417288bc/README.md#sponsors) - would you mind taking a quick look and letting me know if there's anything in the copy you'd like to adjust?

**Review by whummer — APPROVED** [review]
Looks awesome, thanks @bcodell  for adding the sponsor section! 🙌

**Pull request merged**